window_switcher.py

```diff
@@ -14,20 +14,23 @@
 
 __iid__ = "PythonInterface/v0.1"
 __prettyname__ = "Window Switcher"
-__version__ = "1.4"
+__version__ = "1.5"
 __author__ = "Ed Perez, Manuel Schneider"
 __dependencies__ = ["wmctrl"]
 
 if which("wmctrl") is None:
     raise Exception("'wmctrl' is not in $PATH.")
 
+def matchWindow(win, text):
+    return (text in win.wm_class.lower() or text in win.wm_name.lower())
```

Comment:
please no single line functions. This does not really improve maintainability/readability.